root/report/AnnotationsArtists.js

```diff
@@ -32,9 +32,17 @@ const AnnotationsArtists = ({
       <li>
         {l('This report lists artists with annotations.')}
       </li>
+      <li>
+        {l(`If you see something in these annotations that can be represented
+            with a relationship instead, please add a relationship and remove
+            that part of the annotation. If something is marked as
+            "sub-optimal", consider checking if a better way to store that
```

Comment:
`“”` should probably be preferred so as to follow [STYLE guidelines](https://musicbrainz.org/doc/Style/Miscellaneous#Use_of_special_characters).

Reply:
resolved in https://github.com/metabrainz/musicbrainz-server/pull/947/commits/69a313ae52c9763cf3117a9e45c71a1a2918448d